hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -502,6 +503,14 @@ private FlinkOptions() {
       .defaultValue(true) // default true for MOR write
       .withDescription("Schedule the compaction plan, enabled by default for MOR");
 
+  public static final ConfigOption<String> COMPACTION_SCHEDULE_SEQ = ConfigOptions
+      .key("compaction.schedule.seq")
+      .stringType()
+      .defaultValue(FlinkCompactionConfig.SEQ_LIFO) // default LIFO  to improve compact stability
+      .withDescription("Compaction plan execution sequence, two options are supported:\n"
+      + "1). FIFO: execute the oldest plan first;\n"
+      + "2). LIFO: execute the latest plan first, by default LIFO");
```

Comment:
Is this necessary ? We only check the `REQUESTED` instant file and if the offline compaction job sees an `INFLIGHT` compaction, it should skip the compaction then, and vice versa.

Reply:
In my case. there leave one-day compaction plans not executed for some other reason and my offline compaction job can't be ready for some reason such as authority authentication. So I just depend on inline compaction (increase the parallelism or memory) but it always failed because of too much compaction plan needed to execute. so maybe it can execute the latest one and the others just  wait offline job to be ready

Reply:
Then we can add a protection here: when there are more than some number of pending compaction commits (> n),
stop executing new plan if the number of pending plans are not reduced.

Anyway, figuring out new executing strategy is a more proper way.

Reply:
or when some number of pending compaction commits (> n) we can change the execution sequence  dynamic such as from FIFO to LIFO